healthcareai/develop_supervised_model.py

```diff
@@ -13,6 +13,7 @@
 from sklearn.neighbors import KNeighborsClassifier
 from sklearn.linear_model import SGDClassifier
 
+import healthcareai.common.helpers
```

Comment:
I would import this the same way the others are imported:
`from healthcareai.common import helpers`

Then in the code that uses this, you would reference `helpers` instead of `healthcareai.common.helpers`